### PR TITLE
Updates to correctly reference the default branch after renaming.

### PR DESCRIPTION
**Pull request by WadeBarnes**
- GitHub will automatically redirect links to the old branch name, however the URLs should eventually be updated once all of the repositories have been updated.

Signed-off-by: Wade Barnes <wade.barnes@shaw.ca>

### CODE REVIEW

**Comment by WadeBarnes** [comment]
@swcurran, @andrewwhitehead, Please review, but do not merge.  I'll merge once the branch has been renamed.

**Review by andrewwhitehead — APPROVED** [review]

**Pull request merged**
